Android.bp

```diff
@@ -0,0 +1,118 @@
+cc_defaults {
+    name: "maesdk_defaults",
+    cflags: [
+        "-DBUILD_SHARED_LIBS=1",
+        "-DUSE_ROOM=1"
+    ],
+    cppflags: [
+        "-fexceptions",
+        "-Wno-missing-field-initializers",
+        "-Wno-non-virtual-dtor",
+        "-Wno-reorder-ctor",
+        "-Wno-unused-const-variable",
+        "-Wno-unused-parameter",
+        "-Wno-unused-variable"
+    ],
+    rtti: true,
+    vendor_available: true
+}
+
+cc_library_shared {
+    name: "libmaesdk",
+    defaults: ["maesdk_defaults"],
+    srcs: [
```

Comment:
Is there something in there that can't be included using `**/*` - I don't know, maybe `lib/api/**/*.cpp` would've worked. Totally realize that explicitly listing each item is more explicit than adding all by mask.

Reply:
That might also work. We were trying to replicate how the Cmakelist selectively chooses .cpp files based on the build configuration. I don't think we (myself or the original folks who set up our builds in Surface) ever tested using a wildcard and investigated the library output.